CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - Changed type of `errors` in `PaynowException` to `Error`
 - Changed the name of method `getErrorType` in `Error`
 
+## 1.0.6
+- Marked `getErrorType` as deprecated since 2.0.0
```

Comment:
this indicates that getErrorType will be marked as depricated from 2.0.0 and removed in 3.0.0

Reply:
Should I mark it as deprecated in 1.0.6 and remove it in 2.0.0?

Reply:
long story short: yes